### PR TITLE
Story/gat 1674

### PR DESCRIPTION
**Pull request by ThinkingMonkeyLtd**

### CODE REVIEW

File: src/setupTests.js [hunk 1]
```diff
@@ -95,14 +97,18 @@ const queryClient = new QueryClient({
...
     return (
         <I18nextProvider i18n={i18n}>
             <Suspense fallback='Loading'>
                 <ThemeProvider theme={merge(theme, DEFAULT_THEME)}>
                     <AuthProvider value={{ userState: mockUser.data }}>
                         <QueryClientProvider client={queryClient}>
-                            <CmsProvider>{children}</CmsProvider>
+                            <CmsProvider>
+                                <MemoryRouter history={history}>{children}</MemoryRouter>
+                            </CmsProvider>
```
Comment (kymmeh) [inline src/setupTests.js:111]: The plan is to deprecate this file and to import from 'testUtils' which has a custom render function in (this follows set up as described on the library documentation). Could you make those changes in that file instead, and change the import within the test from '@testing-library/react' to 'testUtils'?

File: src/utils/auth.js [hunk 2]
```diff
@@ -54,3 +56,10 @@ export const isPublisherAdmin = (userState, publisherId) => {
         return publisherId === team._id && team.isAdmin;
     });
 };
+
+export const getRolesList = roles => {
+    const sortedRoles = (roles || []).sort();
+
+    // TODO: GAT-1510:043
+    return sortedRoles.map(role => PERMISSIONS_ROLE_NAMES[role]).join(', ');
+};
```
Comment (kymmeh) [inline src/utils/auth.js:65]: Can we add a test for this?

File: src/services/teams/mockMsw.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            organisation: 'Google',
+            bio: 'Manager',
+        },
+    ],
+};
+const mswGetMembers = rest.get(`${apiURL}/teams/5f7b1a2bce9f65e6ed83e7da/members`, (req, res, ctx) => {
+    return res(ctx.status(200), ctx.json(mockTeamMembers));
+});
+
+export default [mswGetMembers];
```
Comment (kymmeh) [inline src/services/teams/mockMsw.js:28]: Can we move any mockMsw files into the 'tests' folder? We can then start having a reusable set of mocks.

**Review by kymmeh — CHANGES_REQUESTED** [review]

File: src/modules/AccountTeamMembers/AccountTeamMembers.jsx [hunk 4]
```diff
@@ -66,96 +78,72 @@ const AccountTeamMembers = ({ userState, teamId }) => {
     return (
         <>
             <LayoutContent>
```
Comment (kymmeh) [inline src/modules/AccountTeamMembers/AccountTeamMembers.jsx:80]: Do we still need LayoutContent?

File: src/utils/auth.js (hunk 2 above)
Comment (ThinkingMonkeyLtd) [reply src/utils/auth.js:65]: Done

Comment (ThinkingMonkeyLtd) [reply src/utils/auth.js:65]: Done

File: src/setupTests.js (hunk 1 above)
Comment (ThinkingMonkeyLtd) [reply src/setupTests.js:111]: Done

File: public/locales/en-GB/translation.json [hunk 5]
```diff
@@ -476,6 +484,14 @@
     "components": {
         "QualityScore": {
             "tooltip": "<p>Metadata richness score: {{score}}</p><p>The score relates to the amount of information available about the dataset, and not to the quality of the actual datasets.</p><p>Click to read more about how the score is calculated.</p><p>{{completeness_percent}} Weighted completeness %<br />{{error_percent}} Weighted error %</p>"
+        },
+        "AccountMembers": {
```
[the remark this answers is not in the document]
Comment (ThinkingMonkeyLtd) [reply public/locales/en-GB/translation.json]: Done

**Review by kymmeh — APPROVED** [review]

**Pull request merged**